addons/hr_timesheet/views/hr_timesheet_views.xml

```diff
@@ -211,7 +211,7 @@
                         <t t-name="kanban-box">
                             <div class="oe_kanban_global_click o_kanban_record_has_image_fill px-0 pb-0">
                                 <div class="oe_kanban_details d-flex flex-column">
-                                    <div class="o_kanban_record_top px-2 h-50">
+                                    <div class="o_kanban_record_top px-2 h-51">
```

Comment:
I change o_kanban_record_top height because adding so_line field in kanban view doesn't display kanban view top properly some part of so_line field value display in bottom part of view [see this](https://drive.google.com/file/d/16i0UW72NQvz-HrlHUNI8bTznJiTUnb4R/view?usp=sharing)